use-cases.md

```diff
@@ -125,3 +125,24 @@ Mobile Key solutions require a state of reservation in real time, a key should n
 
 After receiving a websocket event, use [Get all reservations by ids](operations/reservations.md#get-all-reservations-by-ids) to retrieve information about the reservation and customer if the websocket event fits your criteria. With this response, you will have information to issue the mobile key to the customer using their contact information in Mews.
 
+## Event Management Software
```

Comment:
Why each word has a capital first letter? Why not just Event management software? Same bellow